src/main/java/Display.java

```diff
@@ -14,32 +14,62 @@ public void showGrid(Grid grid) {
                         grid.getSquare(6) + " | " + grid.getSquare(7) + " | " + grid.getSquare(8);
         consoleWriter.println(squaresOutput);
     }
-
+    
     public void makeMoveMessage() {
         var output = "Choose a position from 1 - 9:";
         consoleWriter.println(output);
     }
-
-    public void invalidMoveMessage() {
-        var output = "Invalid move. Please try again.";
+    
+    public void invalidMoveMessage(String mark) {
+        var output = "Invalid move. Please try again, " + mark + ".";
         consoleWriter.println(output);
     }
-
+    
     public void outcomeMessage(String outcome) {
         if (outcome.equals("tie")) {
```

Comment:
What would be the pro's and con's of making "tie" an enum?

Reply:
I'll look into this. Thanks, Ashley